tests/test_json_output.py

```diff
@@ -0,0 +1,27 @@
+from modules.Connection import Connection
+from BaseCollector import BaseCollector
+
+
+class TestJsonOuput:
+    def test_yaml_read(self, rest_yaml):
+        bc_data = BaseCollector.read_rest_yaml()
+        assert rest_yaml == bc_data
+
+    def test_connection_get(self, setup_vcenter, rest_yaml, vmon_json):
+        api_target = rest_yaml['vmonservice']['api_target']
+        con = Connection(setup_vcenter)
+        response = con.get_request(api_target)
+        assert response is False
+        con.login()
+        response = con.get_request(api_target)
+        assert vmon_json == response
+
+    def test_connection_post(self, setup_vcenter, rest_yaml, logging_json):
+        api_target = rest_yaml['logging']['api_target']
+        data = rest_yaml['logging']['action']
+        con = Connection(setup_vcenter)
+        response = con.post_request(api_target, data)
+        assert response is False
```

Comment:
Here I would add some information why False is expected (missing login).

Reply:
I implemented custom error messages.